meta-xt-prod-devel-rcar-control/conf/layer.conf

```diff
@@ -0,0 +1,12 @@
+# We have a conf and classes directory, add to BBPATH
```

Comment:
Looks like you accidentally added the whole meta-xt-prod-devel-rcar-control layer. Could you please remove it from the initial commit?